infrastructure/template.yaml

```diff
@@ -103,6 +104,29 @@ Conditions:
     - production
 
 Resources:
+  JwtSignerFunction:
+    Type: AWS::Serverless::Function
+    Metadata:
+      BuildMethod: esbuild
+      BuildProperties:
+        Minify: true
+        Target: "es2020"
+        Sourcemap: true
+    Properties:
+      Handler: lambdas/jwt-signer/src/jwt-signer-handler.lambdaHandler
+      CodeSigningConfigArn:
+        !If [EnforceCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
+      FunctionUrlConfig:
+        AuthType: AWS_IAM
+        Cors:
```

Comment:
Do we want cors enabled for this lambda? Any reason it's not being created like the other functions?

Reply:
This was generated by the cri-template, i suppose it should reflect the minimium baseline required